### PR TITLE
fedekunze/1131 show only err msgs on notifications

### LINKED ISSUE #1131: Parse LCD errors in the lcdClient
UI Version: 0.9.4

Description:

The LCD delivers errors as some complex string that includes the actual error message. We want to show only the final error to the user. In delegation we are parsing this error already once (#1094). But it would be nice, to parse any error that is delivered in that format and only return that error message to the rest of the system, abstracting the error format away from the Voyager logic.

### PR DESCRIPTION
**Pull request by fedekunze**
Closes #1131

_Description:_

❤️ Thank you!

---

* [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
* [ ] Reviewed `Files changed` in the github PR explorer

### CODE REVIEW

File: app/src/main/index.js [hunk 1]
```diff
@@ -82,14 +82,14 @@ function sleep(ms) {
   return new Promise(resolve => setTimeout(resolve, ms))
 }
 
-function handleCrash(error) {
+function handleCrash(err) {
```
Comment (faboweb) [inline app/src/main/index.js]: going from a non abbreviated word to an abbreviation feels weird.

**Comment by faboweb** [comment]
Do you think this is an advantage? I think the main problem was showing the correct errors from the LCD and not sth like `500 The server produced and error`.

File: app/src/main/index.js (hunk 1 above)
Comment (fedekunze) [reply app/src/main/index.js]: `error` is a reserved word in some programming languages. Also it makes it easier to search retrieved error responses

**Comment by fedekunze** [comment]
> Do you think this is an advantage? I think the main problem was showing the correct errors from the LCD and not sth like 500 The server produced and error.

The correct errors from the LCD are fixed on the SDK. The missing part was verifying that we didn't append any additional message on top of them

**Pull request merged**